src/sas/qtgui/Calculators/media/shape_2_sas_help.rst

```diff
@@ -0,0 +1,87 @@
+.. shape_2_sas_help.rst
+
+.. by J Krzywon, NIST, May 2025
+
+.. _Shape2SAS_Tool:
+
+Shape2SAS
+=========
+
+Description
+-----------
+
+This tool allows the user to build particles from a series of predefined shapes. By allowing particles to be placed in
+any location, or orientation relative to an origin, many unique structures can be constructed. Once a structure is
+created, the user can generate a theoretical data set that can be analysed, or an empirical fitting model can be
+generated to compare the structure to real data.
+
+Please note, the particles are constructed from a series of 'atoms' randomly distributed within the defined shape.
+Because of this randomness, creating the same particle two different times may result in subtle differences between
+scattering curves and slight differences in the resulting empirical models.
+
+The base shapes available to the user include spheres, cylinders, ellipsoids, elliptical cylinders, disks, cubes, cuboids,
+hollow spheres, hollow cubes, and cylindrical rings. Each of the particles can be individually modified in all dimensions,
+can be shifted relative to the origin, rotated relative to the particle centroid, and rotated relative to the coordinate
+volume origin. The scattering length density, relative to the medium the particle is in, can also be defined. If the
+particle is in air, this should be considered the scattering length density of the particle material.
+
+Using the Tool
+--------------
+
+.. image:: main_window.png
+
+When the tool is first selected from the menu option, the 'Build Model' tab will open. The user can select a shape and
+'Add' it to the table in the center of the window. The first row of the table is the particle shape, the second row is
+the scattering length density relative to the solvent, the next three rows are particle dimensions which vary between
+shapes, the sixth through eighth rows are to position the particle in space, the ninth through eleventh rows are for
+particle rotation about the particle centroid, and 12th through 15th rows are to rotate the particle relative to the
+coordinate origin, and the last row defines the particle color.
+
+Once a series of shapes is defined, the 3D rendering and predicted SAS scattering can be calculated by clicking Plot.
+The 'Include Scattering' checkbox must be selected for the SAS data calculation. If the resulting image does not look
+as expected, modify the shape table and rerun the calculation until everything is as desired. By default, any overlap in
+shapes is ignored in the shape construction and scattering calculation. To change this, uncheck the 'Exclude overlap'
+option to account for any differences in SLD in the overlap region and visualize the particle overlap.
```

Comment:
I don't understand what exactly this does either. So if I "exclude" points in the overlap does that remove all points in the overlap leaving a void? Or does it only exclude points from one of the shapes? If so which one? and if I do NOT exclude the points what is calculated? A volume with twice the point density and an effective SLD that is the average of the two shapes? This also needs more clarity, at least for me.